1.4/Defs/AbilityDefs/Archotechist.xml

```diff
@@ -1,335 +1,344 @@
+        <level>2</level>
+        <order>2</order>
+        <psyfocusCost>0.08</psyfocusCost>
+        <entropyGain>0</entropyGain>
+        <prerequisites>
+          <li>VPE_Burden</li>
+          <li>VPE_Stun</li>
+        </prerequisites>
+      </li>
+      <li Class="VanillaPsycastsExpanded.AbilityExtension_TransferEntropy">
+        <targetReceivesEntropy>true</targetReceivesEntropy>
+      </li>
+    </modExtensions>
+  </VFECore.Abilities.AbilityDef>
+
+  <VFECore.Abilities.AbilityDef ParentName="VPE_PsycastBase">
+    <defName>VPE_Berserk</defName>
+    <label>berserk</label>
+    <description>Induce a berserker psychosis in the target's mind, causing them to attack anyone nearby. They regain rationality after 15 seconds, depending on their psychic sensitivity.</description>
+    <iconPath>Abilities/Archotechist/Berserk</iconPath>
+    <abilityClass>VFECore.Abilities.Ability_Blank</abilityClass>
+    <castTime>15</castTime>
+    <durationTime>900</durationTime>
+    <range>20</range>
+    <targetingParameters>
+      <canTargetSelf>True</canTargetSelf>
+      <canTargetHumans>True</canTargetHumans>
+      <canTargetMechs>True</canTargetMechs>
+      <canTargetAnimals>True</canTargetAnimals>
+      <neverTargetIncapacitated>True</neverTargetIncapacitated>
+    </targetingParameters>
+    <goodwillImpact>-15</goodwillImpact>
+    <modExtensions>
+      <li Class="VanillaPsycastsExpanded.AbilityExtension_Psycast">
+        <path>VPE_Archotechist</path>
+        <level>3</level>
+        <order>1</order>
+        <psyfocusCost>0.04</psyfocusCost>
+        <entropyGain>40</entropyGain>
+        <psychic>true</psychic>
+        <prerequisites>
+          <li>VPE_PsychicShock</li>
+        </prerequisites>
+      </li>
+      <li Class="VanillaPsycastsExpanded.AbilityExtension_GiveMentalState">
+        <stateDef>Berserk</stateDef>
+        <stateDefForMechs>BerserkMechanoid</stateDefForMechs>
+        <durationMultiplier>PsychicSensitivity</durationMultiplier>
+      </li>
+    </modExtensions>
+  </VFECore.Abilities.AbilityDef>
+
+  <VFECore.Abilities.AbilityDef ParentName="VPE_PsycastBase">
+    <defName>VPE_AggressiveHeatDump</defName>
+    <label>aggressive heat dump</label>
+    <description>Instantly dump all of your neural heat into a target, causing them to explode with an area of effect relative to the heat lost and the caster’s psychic sensitivity. A good declaration of war.</description>
+    <iconPath>Abilities/Archotechist/AggressiveHeatDump</iconPath>
+    <targetingParameters>
+      <canTargetLocations>False</canTargetLocations>
+      <canTargetBuildings>False</canTargetBuildings>
+      <canTargetHumans>True</canTargetHumans>
+      <neverTargetIncapacitated>True</neverTargetIncapacitated>
+    </targetingParameters>
+    <abilityClass>VanillaPsycastsExpanded.Ability_AggressiveHeatDump</abilityClass>
+    <castTime>60</castTime>
+    <goodwillImpact>-200</goodwillImpact>
+    <range>25</range>
+    <radius>1</radius>
```

Comment:
Could you change this to be the same as the default flame damage?

Reply:
Sure. Would a comment explaining it be okay so the logic is still known and it's not a magic number?

Reply:
Yeah that makes a lot of sense.